build_docker.sh

```diff
@@ -133,3 +141,6 @@ else
 	echo "Not pushing to dockerhub"
 fi
 
+cd ${ORIGINAL_WORKING_DIRECTORY}
+rm -Rf ${STAGING_DIR}/${STAGING_CLONE_DIR}
```

Comment:
Can do, Also it looks like my branches merged together when I did not intend them to. Either way I shall add this.